bot/modules/community/commands.js

```diff
@@ -233,7 +233,7 @@ exports.changeVisibility = async ctx => {
     if (!(await validateObjectId(ctx, id))) return;
     const community = await Community.findOne({
       _id: id,
-      creator_id: ctx.user._id,
+      creator_id: ctx.user._id
```

Comment:
[as you can read in our contribution section, we are using the airbnb javascript guide style, please don't remove the comma

](https://github.com/airbnb/javascript?tab=readme-ov-file#commas--dangling)